pytorch/training/docker/1.10/py3/cu113/Dockerfile.sagemaker.gpu

```diff
@@ -51,12 +50,19 @@ COPY start_with_right_hostname.sh /usr/local/bin/start_with_right_hostname.sh
 WORKDIR /root
 
 RUN pip install --no-cache-dir -U \
-    smdebug==${SMDEBUG_VERSION} \
+    # smdebug==${SMDEBUG_VERSION} \
     smclarify \
     "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
     sagemaker-pytorch-training
 
+# Install smdebug from souce for test purpose
+RUN cd /tmp \
+  && git clone -b 1.0.13 https://github.com/awslabs/sagemaker-debugger \
```

Comment:
I am seeing that the `SMDEBUG_VERSION` used above is 1.0.12, however we are checking out to 1.0.13, is that on purpose?

Reply:
SMDEBUG team suggested to make such change. smdebug 1.0.12 does not support PT1.10.2, upgrading to 1.0.13 for the support